clusterloader2/pkg/framework/client/objects.go

```diff
@@ -64,6 +64,8 @@ func IsRetryableAPIError(err error) bool {
 	// These errors may indicate a transient error that we can retry in tests.
 	if apierrs.IsInternalError(err) || apierrs.IsTimeout(err) || apierrs.IsServerTimeout(err) ||
 		apierrs.IsTooManyRequests(err) || utilnet.IsProbableEOF(err) || utilnet.IsConnectionReset(err) ||
+		// 409 error may be returned in some cases, e.g. https://github.com/kubernetes/kubernetes/issues/67761
+		apierrs.IsConflict(err) ||
```

Comment:
This one is fairly tricky - not every conflict will be automatically retryable.
I'm slightly afraid of doing that automatically for every case.

Instead of that, we may allow that explicitly in specific calls...